fpga_interchange/chipdb.h

```diff
@@ -306,6 +328,14 @@ inline const SiteInstInfoPOD &site_inst_info(const ChipInfoPOD *chip_info, int32
     return chip_info->sites[chip_info->tiles[tile].sites[site]];
 }
 
+enum SyntheticType {
+    NOT_SYNTH = 0,
+    SYNTH_SIGNAL = 1,
```

Comment:
is there a particular use case intended for non-gnd/vcc synthetic signals, out of curiosity?

Reply:
One of the techniques we’ve been working on is static fabric subregions, which is related to out of context builds.  We might use synth signals to represents those